.changeset/neat-sheep-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/scaffolder-backend-module-utils': patch
```

Comment:
Is this a breaking change? It introduces an underlying command `yq` which is not one of the default ones in any sane distros

Reply:
My thinking here is that you dont have to choose to use this action and current usage is not using it. So it is not breaking in that sense. Happy to make it breaking if you have a different perspective.